derive/src/pyclass.rs

```diff
@@ -337,13 +337,56 @@ where
 
         let item_attr = args.attrs.remove(self.index());
         let item_meta = MethodItemMeta::from_attr(ident.clone(), &item_attr)?;
-
         let py_name = item_meta.method_name()?;
+
+        let sig_doc = args.item.function_or_method_impl().ok().map(|item| {
+            let sig = item.sig();
+            let mut has_args = false;
+            let mut args: Vec<_> = sig
+                .inputs
+                .iter()
+                .filter_map(|arg| {
+                    use syn::FnArg::*;
+                    let arg = match arg {
+                        Receiver(_) => return Some("$self".to_owned()),
+                        Typed(typed) => typed,
+                    };
+                    let ty = arg.ty.as_ref();
+                    let ty = quote!(#ty).to_string();
+                    if ty == "FuncArgs" {
+                        has_args = true;
```

Comment:
`'/'` is used to separate positional-only parameters from positional-or-keyword ones (similarly `*` separates positional_or_keyword parameters from keyword only). I.e the `'/'` separator should be placed before the first `Optional` is encountered (or at the end if the function only has positionals). 

Since this is a first step, I say we remove `'/'` for now and fix it up when we cover more cases.